data/human/human missions.txt

```diff
@@ -7453,3 +7453,405 @@ mission "Adelita 5"
+			choice
+				`	"Of course."`
+					goto next1
+				`	"Didn't you message me as soon as I did it?"`
+			`	"True enough, just making conversation," he says with a grin. "The important thing is that the job's done. Now, let's get down to business.`
+			label next1
+			`	"Our recovery team is already retrieving the ship as we speak. You've done well. Here's a payment for that favor, 25,000 credits."`
+			choice
+				`	"Thanks."`
+					goto investmentopps
+				`	"25,000?! You must have forgotten a zero there, you cheapskate!"`
+					goto cheapskate
+				`	"I appreciate the offer, but there's no need for payment this time. Consider it purely a favor--I'm happy to help."`
+			action
+				"syndicate business: mean to Howard" --
+				payment -25000
+			`	"That's generous of you, and I appreciate it."`
+				goto investmentopps
+			label cheapskate
+			`	"I see... well, let's not make this more difficult than it needs to be," he says in a clipped tone. "I'll transfer another 175,000 credits now to make it an even 200,000. But in the future, I'd appreciate it if you would remember the privilege you have of working with Megaparsec Incorporated.`
+			action
+				"syndicate business: mean to Howard" ++
+				payment 175000
+			label investmentopps
+			`	"Now, let's talk about that investment opportunity."`
+			`	Howard pulls up a holographic display on the digital screen behind him. Various charts and projections flash by. "Ready for the pitch?"`
+			choice
+				`	"You know it."`
+				`	"Of course."`
+			`	"Here's what we're looking at. Megaparsec Incorporated is expanding a refueling station on Crossroads, and we're likely going to go with a conventional expansion. But don't let the term 'conventional' fool you, because Megaparsec rarely expands its refueling stations--this is the first time in over a decade. When we do, it's because we've identified a strategic location that will skyrocket profits. These opportunities are as rare as they come, and most investors never even hear about them. But you've proven yourself, and that's why you're getting a seat at the table."`
+			`	He pauses, letting the weight of his words sink in. "Getting in on this is a once-in-a-lifetime chance. We're talking about a key hub on a high-traffic trade route, with a captive market of ships desperate for fuel. The returns will be solid... and dependable. Opportunities like this don't just come around, and when they do, they go fast."`
+			choice
+				`	"Okay."`
+			branch upcharge
+				"syndicate business: mean to Howard" > 0
+			`	"Excellent. The price for this investment is ten million credits for a solid 2700 credit per day return. Don't worry about pulling out your wallet just yet--I've pulled a few strings with our friends at the bank. They're willing to structure a loan for you through our Finance Division, no upfront payment needed. You'll just need to sign a few documents to get things moving."`
```

Comment:
I suppose one question would be why you don't have a choice to take a bank loan or not.
You might want to pay out-of-hand.

Reply:
From a gameplay perspective it doesn't _really_ matter since the player can pay out-of-hand by just immediately paying off the loan, but if it's a bit jarring that you don't have that option in the conversation I can add it in.

Reply:
I'm not sure.
It probably makes sense for Howard to not give you the option to pay out of hand - he's already "pulled some strings" to offer you this, the bank is unlikely to appreciate him canceling on them.
And yes, seeing as how it doesn't impact gameplay it's probably fine to not change it.